crates/ruff_linter/src/rules/airflow/rules/dag_schedule_argument.rs

```diff
@@ -0,0 +1,113 @@
+use ruff_diagnostics::{Diagnostic, Violation};
+use ruff_macros::{derive_message_formats, violation};
+use ruff_python_ast::Expr;
+use ruff_python_ast::{self as ast, Keyword};
+use ruff_text_size::Ranged;
+
+use crate::checkers::ast::Checker;
+
+/// ## What it does
+/// Checks that the `DAG()` class or a `@dag()` decorator has an explicit
+/// `schedule` parameter.
+///
+/// ## Why is this bad?
+/// The default `schedule` value on Airflow 2 is `timedelta(days=1)`, which is
+/// almost never what a user is looking for. Airflow 3 changes this the default
+/// to *None*, and would break existing DAGs using the implicit default.
```

Comment:
Can we tell a bit more about *how* it breaks existing DAGS that use the implicit default. Will it fail with an exception or is it just that the default is different?

Reply:
Just the default is different—your DAGs will simply stop doing anything silently. I’ll add a paragraph to call this out.